### ISSUE TITLE
another file format

### ISSUE THREAD

**Issue by syphzur** [JChrobak/Fajne-Studia-2020#1]
Please add suport for files in another format (question, answer, T/F). Example:
Dla unikniecia hazardu danych mozna Zastosowac uproszczone metody adresacji.
RISC nie eliminuje hazardu XD Skracaja tylko czas dostepu do pamieci.
F

**Comment by mateuszwende** [comment]
I fully agree with @syphzur. This is very neede feature, which we people need right now. #AFLM
